Index: src/main/webapp/pages/test2.jsp
===================================================================
--- src/main/webapp/pages/test2.jsp	(revision 5a1791335b88333973bdc2dd7793b6ee21c2df18)
+++ src/main/webapp/pages/test2.jsp	(revision 5a1791335b88333973bdc2dd7793b6ee21c2df18)
@@ -1,16 +0,0 @@
-<%--
-  Created by IntelliJ IDEA.
-  User: tianxi
-  Date: 16-3-17
-  Time: 下午4:54
-  To change this template use File | Settings | File Templates.
---%>
-<%@ page contentType="text/html;charset=UTF-8" language="java" %>
-<html>
-<head>
-    <title>Title</title>
-</head>
-<body>
-
-</body>
-</html>
